.php_cs

```diff
@@ -5,7 +5,6 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 ;
 
 return Symfony\CS\Config\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
Another solution would be to use the same solution used in Php-cs-fixer (a regex replacement).